packages/fiber/src/core/utils.ts

```diff
@@ -333,8 +346,7 @@ export function applyProps(instance: Instance, data: InstanceProps | DiffSet) {
         // For versions of three which don't support THREE.ColorManagement,
         // Auto-convert sRGB colors
         // https://github.com/pmndrs/react-three-fiber/issues/344
-        const supportsColorManagement = 'ColorManagement' in THREE
-        if (!supportsColorManagement && !rootState.linear && isColor) targetProp.convertSRGBToLinear()
+        if (!ColorManagement && !rootState.linear && isColor) targetProp.convertSRGBToLinear()
```

Comment:
What do you think about putting this condition in a variable?

Type `const condition = !ColorManagement && !rootState.linear && isColor`

But using an appropriate nomenclature